### ISSUE TITLE
Diagnostic Notes

### ISSUE THREAD

**Issue by foneworkshop** [Repair-DeskPOS/RepairDesk-BUGS-IMPROVEMENTS#351]
When I open a ticket AFTER it has been booked in through manage tickets, there is a box for diagnostic note and a flag.  When I enter notes and save the notes disappear. Nothing different if I flag it.

**Comment by Repair-Desk** [comment]
**@usmanalikhan93**  

Can you please check if you add a diagnostic note from manage tickets page and re-open the ticket on point of sale it automatically display the diagnostic notes.

Usman

**Comment by foneworkshop** [comment]
Tried that,  there is nothing showing on pos

**Comment by usmanalikhan93** [comment]
@Repair-Desk 
@foneworkshop

Sure thing! Let me take a look to see if I'm getting the same issue as you and I'll circle back to you.

usman ali

**Comment by usmanalikhan93** [comment]
Hey @foneworkshop 👋 
Thank you for bringing this to our attention! It looks like there's a bug that just popped up 🐛. 

I'm gonna reach out to our Engineers to get this taken care of, I'll get back to you as soon as I have more information! 😃
usman ali

**Comment by usmanalikhan93** [comment]
Hey @foneworkshop !

Can you check this video and let us know, you are following these steps for adding diagnostic notes ?
https://www.screencast.com/t/OhF2AYeUvpq

usman ali

**Comment by foneworkshop** [comment]
Hi,



This is not what I asked,  I do not want POS I want to add and flag notes
AFTER booking in from manage tickets

 

 

 

 

-------Original Message-------

 

From: Usman Ali Khan

Date: 28/06/2018 06:42:35

To: Repair-DeskPOS/RepairDesk-Bugs

Cc: foneworkshop;  Mention

Subject: Re: [Repair-DeskPOS/RepairDesk-Bugs] Diagnostic Notes (#351)

 

Hey @foneworkshop !

Can you check this video and let us know, you are following these steps for
adding diagnostic notes ?

https://www.screencast.com/t/OhF2AYeUvpq

usman ali

—

You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub, or mute the thread.

 

---
This email has been checked for viruses by AVG.
https://www.avg.com

**Comment by Repair-Desk** [comment]
**Hello @foneworkshop,**

Thanks for sharing more details. 

I've discussed this with our product team and currently flagged diagnostic notes pop up doesn't display on the view tickets page.

I've pinned the improvement to our roadmap and hopefully we will be able to look into this in next couple of weeks.

And I hope that if there’s anything else I can do for you, you won’t hesitate to let me know.

Usman
